modules/messages/client/less/thread.less

```diff
@@ -16,11 +16,11 @@
 
   @media (min-width: @screen-xs-min) {
     .content-empty {
-      position: absolute;
-      bottom: 50%;
-      margin-top: -65px;
-      left: 0;
-      right: 0;
+      // position: absolute;
```

Comment:
I'm not sure what these styles were originally for. It didn't make a visible different after it was disabled.